cmd/ddev/cmd/addon-list.go

```diff
@@ -0,0 +1,137 @@
+package cmd
+
+import (
+	"bytes"
+	"fmt"
+	"sort"
+	"strings"
+
+	"github.com/ddev/ddev/pkg/ddevapp"
+	"github.com/ddev/ddev/pkg/globalconfig"
+	"github.com/ddev/ddev/pkg/output"
+	"github.com/ddev/ddev/pkg/styles"
+	"github.com/ddev/ddev/pkg/util"
+	"github.com/google/go-github/v52/github"
+	"github.com/jedib0t/go-pretty/v6/table"
+	"github.com/jedib0t/go-pretty/v6/text"
+	"github.com/spf13/cobra"
+)
+
+// AddonListCmd is the "ddev add-on list" command
+var AddonListCmd = &cobra.Command{
+	Use:   "list",
+	Short: "List available or installed DDEV add-ons",
```

Comment:
and one nitpick i havent looked at all the help texts yet. but list available addons is correct but i am unsure if "installed" addons are listed too. list just lists official addons and with the all flag also unofficial one but installed addons are not have any additional label in the outputted list?